.github/actions/setup/action.yml

```diff
@@ -0,0 +1,38 @@
+name: "Variables setup"
+description: "Variables setup"
+
+runs:
+  using: "composite"
+  steps:
+    - uses: Alfresco/alfresco-build-tools/.github/actions/get-commit-message@v1.35.0
+    - name: setup variables
+      shell: bash
+      run: |
+        echo "BUILD_OPTS=--configuration=production,e2e" >> $GITHUB_ENV
+        echo "TEST_OPTS=" >> $GITHUB_ENV
+        echo "E2E_PROTRACTOR_OPTS=" >> $GITHUB_ENV
+        echo "E2E_TSCONFIG=tsconfig.e2e.json" >> $GITHUB_ENV
+        echo "GIT_HASH=$(git rev-parse HEAD)" >> $GITHUB_ENV
+        echo "SMART_RUNNER_DIRECTORY=.protractor-smartrunner" >> $GITHUB_ENV
+        echo "BASE_HASH=.protractor-smartrunner" >> $GITHUB_ENV
+        echo "HEAD_HASH=HEAD" >> $GITHUB_ENV
+    - uses: Alfresco/alfresco-build-tools/.github/actions/get-branch-name@v1.35.0
+    - name: set TAG_NPM
+      shell: bash
+      run: |
+        VERSION_IN_PACKAGE_JSON=$(node -p "require('./package.json')".version)
+        echo "version in package.json=${VERSION_IN_PACKAGE_JSON}"
+        if [[ ${{ env.BRANCH_NAME }} =~ ^master(-patch.*)?$ ]]; then
```

Comment:
you don't really need to use the `${{ env.key }}}` notation here in the body of a run step, just go with plain $BRANCH_NAME